### PR TITLE
Group all trivy files in a new language

### PR DESCRIPTION
**Pull request by DMarinhoCodacy**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (stefanvacareanu7) [inline .vscode/settings.json:1]: This file should be removed

Comment (DMarinhoCodacy) [reply .vscode/settings.json]: Thanks!

**Review by stefanvacareanu7 — APPROVED** [review]

**Comment by lolgab** [comment]
Be careful with this change as it can break Codacy.
First of all, you are using the concept of language to express something like `TrivyFile` which is arguably not a language.
Then you are adding `package.json` to this language and removing from Json which is the natural place it belongs to.

What do you want to achieve exactly?

Are you aware of the process documented here? https://codacy.slite.com/app/docs/UCT91ruiVn9tTh

**Comment by DMarinhoCodacy** [comment]
We need to rethink this approach. Closing this PR.

**Pull request closed without merging**